fastlane/Fastfile

```diff
@@ -245,51 +259,55 @@ platform :android do
+  # @param skip_confirm [Boolean] If true, avoids any interactive prompt
+  #
+  lane :finalize_hotfix_release do |skip_confirm: false|
     ensure_git_branch(branch: '^release/') # Match branch names that begin with `release/`
     ensure_git_status_clean
 
     version = release_version_current
 
     UI.important("Triggering hotfix build for version: #{version}")
-    unless options[:skip_confirm] || UI.confirm('Do you want to continue?')
+    unless skip_confirm || UI.confirm('Do you want to continue?')
       UI.user_error!("Terminating as requested. Don't forget to run the remainder of this automation manually.")
     end
 
     trigger_release_build(branch_to_build: "release/#{version}")
     create_backmerge_pr
   end
 
-  # @param [String] branch_to_build (default: current git branch) The branch to build
-  lane :trigger_release_build do |options|
+  # @param branch_to_build [String] The branch to build. Defaults to the current git branch.
+  #
+  lane :trigger_release_build do |branch_to_build: git_branch|
     buildkite_trigger_build(
       buildkite_organization: 'automattic',
       buildkite_pipeline: 'pocket-casts-android',
-      branch: options[:branch_to_build] || git_branch,
+      branch: branch_to_build,
       pipeline_file: 'release-builds.yml'
     )
   end
 
+  # Builds and uploads a new build to Google Play (without releasing it)
+  #
   # - Uses the current version to decide if this is a beta or production build
   # - Builds the apps for external distribution
   # - Uploads the builds to 'beta' or 'production' Play Store channel (but does not release it)
   # - Creates draft Github release
   #
-  # @option [Boolean] skip_confirm (default: false) If true, avoids any interactive prompt
-  # @option [Boolean] skip_prechecks (default: false) If true, skips android_build_preflight
-  # @option [Boolean] create_gh_release (default: false) If true, creates a draft GitHub release
+  # @param skip_confirm [Boolean] If true, avoids any interactive prompt
+  # @param skip_prechecks [Boolean] If true, skips android_build_preflight
+  # @param create_gh_release [Boolean] If true, creates a draft GitHub release
   #
-  desc 'Builds and uploads a new build to Google Play (without releasing it)'
-  lane :build_and_upload_to_play_store do |options|
+  lane :build_and_upload_to_play_store do |skip_prechecks: false, skip_confirm: false, create_gh_release: false|
     version = version_name_current
     build_code = build_code_current
     is_beta = beta_version?(version)
-    unless options[:skip_prechecks]
+    unless skip_prechecks
       # Match branch names that begin with `release/`
       ensure_git_branch(branch: '^release/') unless is_ci
 
       UI.important("Building version #{version_name_current} (#{build_code_current}) for upload to Google Play Console")
-      unless options[:skip_confirm] || UI.confirm('Do you want to continue?')
+      unless skip_confirm || UI.confirm('Do you want to continue?')
```

Comment:
<table data-meta="generated_by_pr-check">
  <tbody>
    <tr>
      <td>:no_entry_sign:</td>
      <td width="100%" data-sticky="false"><span data-href="https://github.com/Automattic/pocket-casts-android/blob/6a0928aa27197a8a7fec52a5f2f55ad056cabff3/fastlane/Fastfile#L310"></span>Style/IfUnlessModifier: Favor modifier <code>unless</code> usage when having a single-line body. Another good alternative is the usage of control flow <code>&amp;&amp;</code>/<code>||</code>.</td>
    </tr>
  </tbody>
</table>